lib/export_tables_to_big_query.rb

```diff
@@ -105,6 +105,8 @@ def bigquery_data(record, table)
     if table.column_names.include?('geolocation')
       @bigquery_data['geolocation_x'] = record.geolocation.x
```

Comment:
Do we still need these lines?

Reply:
we have a `geolocation` field in the organisation table.

Reply:
it is of type `point` and I guess what we are doing here is to extract the lat and long